lib/livebook_web/live/hub/edit_live.ex

```diff
@@ -44,8 +44,23 @@ defmodule LivebookWeb.Hub.EditLive do
       saved_hubs={@saved_hubs}
     >
       <div class="p-4 md:px-12 md:py-7 max-w-screen-md mx-auto space-y-8">
-        <div>
+        <div class="flex relative">
           <PageHelpers.title text="Edit Hub" socket={@socket} />
+
+          <button
+            phx-click={
+              with_confirm(
+                JS.push("delete_hub", value: %{id: @hub.id}),
+                title: "Delete hub",
+                description: "Are you sure you want to delete this hub?",
+                confirm_text: "Delete",
+                confirm_icon: "close-circle-line"
+              )
+            }
```

Comment:
I assume we have custom confirm modals everywhere else; out of curiosity why are they better than native confirm, ie:

```suggestion
            phx-click={JS.push("delete_hub", value: %{id: @hub.id})}
            data-confirm="Are you sure you want to delete this hub?"
```

?

Reply:
Good question, never heard about `data-confirm`.

Reply:
But this is the native confirm from browser, right? The layout is defined by the browser? If yes, this would be the same layout in every browser.

Reply:
Yeah it’s the good old JS `confirm(message)` thing.

Reply:
The custom confirmation modal is consistent with all other modals, which is a better UX by itself, but more importantly our confirm modal supports "don't show again" when needed. For example, when deleting a cell for the first time we show the confirmation and teach the user that deleted cells end up in the bin, but they can opt-out of showing the modal every time, because it would be annoying :)

Reply:
Good to know, sorry for the noise.